### ISSUE TITLE
Not compatible with URP

### ISSUE THREAD

**Issue by vincismurf** [oculus-samples/Unity-TheWorldBeyond#16]
Could you provide URP version of the shaders?

otherwise all the shaders are magenta. I am muddling thru a conversion ( Ottps://developer.arm.com/documentation/102487/0100/Migrating-custom-shaders-to-the-Universal-Render-Pipeline ) But a experienced graphic dev should be able to do this much faster.

Unity is planning on making URP the default, it would be great to see this project get updated accordingly.

**Comment by vincismurf** [comment]
[URP.zip](https://github.com/oculus-samples/Unity-TheWorldBeyond/files/12910844/URP.zip)
Here is a handful of shaders to get the default scene working.

Note to those using it, 
Select Material, note Render Queue , 
Replace shader with  Custom/<SAMENAME>_URP  
Update Render Queue number

Note I am not a graphics engineer and it isn't the whole repo just an after noon of moving CG to HLSL.

Please have a graphics engineer review before committing , some multi pass logic might be wrong.

**Comment by v23v** [comment]
Yes, I hope the sample project can set URP as the default rendering pipeline

**Comment by jasonmeisel** [comment]
Thanks for the report! We're looking into upgrading the project to URP, but for the time being it's just a limitation of the project.
